<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilePasswordPage</class>
 <widget class="QWidget" name="FilePasswordPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="createPasswordButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>390</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #4A90E2;
    color: white;
    font-family: &quot;Inter&quot;, sans-serif;
    font-size: 16px;
    font-weight: 600;
    border: none;
    border-radius: 10px;
    padding: 8px 16px;
}

QPushButton:hover {
    background-color: #5AA9F4;
}

QPushButton:pressed {
    background-color: #3A78C2;
}

QPushButton:disabled {
    background-color: #A0C4FF;
    color: #E0E0E0;
}</string>
   </property>
   <property name="text">
    <string>Create password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Account created! time to create a file encryption password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>691</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>This is extremly important, do not under any circumstances forget it, as you will lose your encrypted files.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordTextField">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="rePasswordTextField">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Re-enter Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
